src/memory/Board.java

```diff
@@ -0,0 +1,210 @@
+package memory;
+
+import javax.swing.*;
+import javax.swing.border.BevelBorder;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+public class Board implements GameService{
+    GameConstants gameConstants;
+
+    JFrame frame;
+    JPanel panelTitle, panelGrid, panelControl;
+    JButton new_game, solve, about;
```

Comment:
avoid declaring multiple variable in a single line.

Reply:
done